src/Domain/Creators/CylindricalBinaryCompactObject.hpp

```diff
@@ -0,0 +1,297 @@
+      domain::CoordinateMap<
+          Frame::Logical, Frame::Inertial,
+          CoordinateMaps::ProductOf2Maps<CoordinateMaps::Wedge<2>,
+                                         CoordinateMaps::Affine>,
+          CoordinateMaps::CylindricalFlatSide,
+          CoordinateMaps::DiscreteRotation<3>>,
+      domain::CoordinateMap<
+          Frame::Logical, Frame::Inertial,
+          CoordinateMaps::ProductOf2Maps<CoordinateMaps::Wedge<2>,
+                                         CoordinateMaps::Affine>,
+          CoordinateMaps::CylindricalSide,
+          CoordinateMaps::DiscreteRotation<3>>>;
+
+  struct CenterA {
+    using type = std::array<double, 3>;
+    static constexpr Options::String help = {
+        "Grid coordinates of center for Object A, which is at x>0."};
+  };
+  struct CenterB {
+    using type = std::array<double, 3>;
+    static constexpr Options::String help = {
+        "Grid coordinates of center for Object B, which is at x<0."};
+  };
+  struct RadiusA {
+    using type = double;
+    static constexpr Options::String help = {
+        "Grid-coordinate radius of grid boundary around Object A."};
+  };
+  struct RadiusB {
+    using type = double;
+    static constexpr Options::String help = {
+        "Grid-coordinate radius of grid boundary around Object B."};
+  };
+  struct OuterRadius {
+    using type = double;
+    static constexpr Options::String help = {
+        "Grid-coordinate radius of outer boundary."};
+  };
+
+  struct InitialRefinement {
+    using type = size_t;
+    static constexpr Options::String help = {
+        "Initial refinement level. Applied to each dimension."};
+  };
+  struct InitialGridPoints {
+    using type = size_t;
+    static constexpr Options::String help = {
+        "Initial number of grid points in each dim per element."};
+  };
+
+  struct TimeDependence {
+    using type =
+        std::unique_ptr<domain::creators::time_dependence::TimeDependence<3>>;
+    static constexpr Options::String help = {
+        "The time dependence of the moving mesh domain."};
+  };
+
+  struct BoundaryConditions {
+    static constexpr Options::String help = "The boundary conditions to apply.";
+  };
+  template <typename BoundaryConditionsBase>
+  struct InnerBoundaryCondition {
+    static std::string name() noexcept { return "InnerBoundary"; }
+    static constexpr Options::String help =
+        "Options for the inner boundary conditions.";
+    using type = std::unique_ptr<BoundaryConditionsBase>;
+    using group = BoundaryConditions;
+  };
+
+  template <typename BoundaryConditionsBase>
+  struct OuterBoundaryCondition {
+    static std::string name() noexcept { return "OuterBoundary"; }
+    static constexpr Options::String help =
+        "Options for the outer boundary conditions.";
+    using type = std::unique_ptr<BoundaryConditionsBase>;
+    using group = BoundaryConditions;
+  };
+
+  using basic_options =
+      tmpl::list<CenterA, CenterB, RadiusA, RadiusB, OuterRadius,
```

Comment:
The other binary domain changed the way it handled the two objects by having an `Object` class for each that holds the options. I'm not requesting that you change to that, just letting you know so you can decide what you prefer :) Not changing, changing in a later PR, or changing in this PR are all equally reasonable to me :)

Reply:
Here there are only two options (Center and Radius) that are different for the two objects.  Perhaps such a change can be made if/when there are more such options?  One thing that will probably be needed in a future PR is different refinement levels in different Blocks (the other binary domain has this feature).

Reply:
Yep, sounds good :)